src/Akeneo/Pim/Enrichment/Bundle/Command/MigrateToUuid/MigrateToUuidCommand.php

```diff
@@ -122,9 +122,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 );
             }
 
+            $this->success();
             $this->logger->notice('Migration done!', ['migration_duration_in_second' => time() - $startMigrationTime]);
-        } finally {
-            $this->stop();
+        } catch (\Throwable $e) {
+            $this->logger->error('Migration failed!', ['message' => $e->getMessage()]);
```

Comment:
```suggestion
            $this->logger->error('Migration failed!', ['message' => $e->getMessage(), 'exception' => $e]);
```
(this way we'll have the exception stack trace)

Reply:
It will not be an issue for the logger to give exception like that ?

Reply:
nope, it will be normalized as well, see https://github.com/Seldaek/monolog/blob/60ad5183b5e5d6c9d4047e9f3072d36071dcc161/src/Monolog/Formatter/JsonFormatter.php#L170